docs/css/customstyles.css

```diff
@@ -1246,3 +1246,7 @@ h4.panel-title {
     width: 100px;
     height: 100px;
 }
+
+.post-content h2 {
+    display: inline-block;
```

Comment:
If we add the edit link/button inside (instead of after) the h2 element (like the anchorjs stuff), we don't need this.

This can be achieved by using jquery's append method: `$('div.post-content').find('h2').append('....')`

Reply:
I was not sure about adding it inside the element, since it would also appear in the toc. Is that a problem?

Reply:
Good point. Currently the anchorjs-link is also inside - and it seems, it also ends up in the toc. I guess, that's wrong.

I think, this can be avoided, if the toc is generated before the other elements are added... I'll check, if the could work

Reply:
Ok, so it works. The problem is, we have multiple places, where we  use `$( document ).ready(`, so it is not clear directly, in which order they are called. Currently the toc is generated after the anchors are created (because _include/toc.html is included after js/customscripts.js). In order to fix this, I'd suggest to add the toc-call also into js/customscripts.js and integrate your document-ready handler for modifying h2 in there as well. Then we have only one place and the order is pretty clear (anchorjs call is already in js/customscripts.js). Then we can generate the toc first, and then add anchors and after that add the pencil icon.

Btw. the pencil icon I used is this: https://emojipedia.org/pencil/